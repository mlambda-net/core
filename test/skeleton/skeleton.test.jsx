import React from 'react';
import renderer from 'react-test-renderer';
import { expect, test } from 'vitest';
import Wrap from '../util';
import Skeleton from '@mlambda-net/web-core/skeleton/skeleton.jsx';

test('Skeleton render', () => {
  const item = renderer.create(
    <Wrap>
      <Skeleton />
    </Wrap>,
  );
  let tree = item.toJSON();
  expect(tree).toMatchSnapshot();
});
