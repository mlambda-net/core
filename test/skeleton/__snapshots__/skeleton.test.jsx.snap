// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Skeleton render 1`] = `
<div
  className="MuiBox-root css-8atqhb"
>
  Skeleton
</div>
`;
